File: emulinker/src/main/java/org/emulinker/kaillera/model/event/AllReadyEvent.java
package org.emulinker.kaillera.model.event;

import org.emulinker.kaillera.model.KailleraGame;

public class AllReadyEvent implements GameEvent {
  private KailleraGame game;

  public AllReadyEvent(KailleraGame game) {
    this.game = game;
  }

  @Override
  public String toString() {
    return "AllReadyEvent";
  }

  @Override
  public KailleraGame getGame() {
    return game;
  }
}

File: emulinker/src/main/java/org/emulinker/kaillera/model/event/AllReadyEvent.kt
package org.emulinker.kaillera.model.event

import org.emulinker.kaillera.model.KailleraGame

data class AllReadyEvent(override val game: KailleraGame) : GameEvent {
  override fun toString(): String {
    return "AllReadyEvent"
  }
}
